packages/core/admin/admin/src/content-manager/pages/CollectionTypeRecursivePath/index.js

```diff
@@ -16,14 +17,13 @@ import ListViewLayout from '../ListViewLayoutManager';
 
 import ErrorFallback from './components/ErrorFallback';
 
-const cmPermissions = permissions.contentManager;
-
 const CollectionTypeRecursivePath = ({
   match: {
     params: { slug },
     url,
   },
 }) => {
+  const permissions = useSelector(selectAdminPermissions);
```

Comment:
any reason we're not chaining the selector to select exactly what we want?

Reply:
Previously every page/ component/ hook had access to all admin permissions. I think it is a good idea to only give them what they need to know about, but if you don't mind I'd like to do this in another iteration.

Reply:
sgtm